packages/vue-app/template/App.js

```diff
@@ -328,14 +324,14 @@ export default {
       <% if (nuxtOptions.generate.manifest) { %>
       const manifest = await this.fetchStaticManifest()
       const path = this.getRoutePath(route)
-      if (!manifest.routes.includes(decodeURI(path))) {
+      if (!manifest.routes.includes(decode(path))) {
```

Comment:
What if we normalize route once in line before? (decode and strip trailing-slash). This way we ensure consistency and little bit perf for not decoding multiple times...

Reply:
In fact it looks like there was a bug in payload fetching with routerBase that this fixes :see_no_evil: